### DISCUSSION TITLE
docs: is forge required? how to trigger pipeline manually?

### DISCUSSION THREAD

**Discussion by erlangparasu** [woodpecker-ci/woodpecker#4060] [Ask a Question]
docs: is forge required? how to trigger pipeline manually?

**Comment by qwerty287** [comment]
Yes, a forge is required. Pipelines can be manually triggered from the repo's overview.

This question has been asked multiple times already. See https://github.com/woodpecker-ci/woodpecker/discussions/2651 and https://github.com/woodpecker-ci/woodpecker/discussions/3591.

Or do you just want to have this more clear in the docs?

**Reply by erlangparasu** [reply]
for example in jenkins, i can have different git repo on each pipeline (github on pipeline1, gitlab on pipeline2, both running in same jenkins instance). in woodpecker, only one repo provider per woodpacker-server?

**Reply by erlangparasu** [reply]
jenkins ref: https://www.jenkins.io/doc/pipeline/steps/params/git/

**Reply by qwerty287** [reply]
Currently yes. We plan to support multiple forges in the future.
